src/components/Header/index.js

```diff
@@ -178,7 +178,7 @@ const GordonHeader = ({ authentication, onDrawerToggle, onSignOut }) => {
             <MenuIcon className="menu-button-icon" />
           </IconButton>
 
-          <Typography className="title disable-select" variant="h6" color="inherit">
+          <Typography className="header disable-select" variant="h6" color="inherit">
```

Comment:
What was the goal/reason for this change? It doesn't seem related to the other changes. Also, there doesn't appear to be any styles that target the `header` class anywhere in the project. There were styles targeting `title` that would've been applied to this element (see `src/components/Header/header.scss` lines 95-99).

Reply:
@EjPlatzer, The reason was to change the the spacing in my profile here. 

Here is the before picture. 

![](https://user-images.githubusercontent.com/17221652/118873376-b16a6d00-b8b7-11eb-903b-1d8074fc0cf7.png)

and Here is the after picture.

![Screen Shot 2021-06-07 at 11 28 15 AM](https://user-images.githubusercontent.com/60272749/121044356-7e631d00-c783-11eb-96ae-7df77b1e4992.png)

That is what I changed in here. I hope that is clear let me know if you have any other question @EjPlatzer

Reply:
Thanks for the clarification! That makes sense that you're trying to fix the spacing on the header. 

I don't think this is the best way to achieve that goal though. The `title` class that was applied to this component was used to give it this styling (found on lines 95-99 of `src/components/Header/header.scss`):
```css
.title {
    flex: 1;
    margin-right: 10px;
    font-weight: bold;
}
```
I think your intent was to remove the `font-weight: bold` property, since a bold font takes up more space than a regular font. However, by removing the entire `title` class, you also removed the other properties - most notably `flex: 1`. That has some unintended effects, especially when you view the page on a mobile or tablet sized screen (less than 960 pixels wide):
<img width="958" alt="Screen Shot 2021-06-07 at 3 13 50 PM" src="https://user-images.githubusercontent.com/35319956/121075269-f987fb80-c7a2-11eb-9181-8dac06c7263f.png">

I think the best approach would be to use the original `title` class, and update `header.scss` so that `title` is not bold.

Also, this doesn't seem to be relevant to the other changes you made in this pull request. Maybe you should move them to a separate branch and submit a different pull request focused on this issue?